frontend/src/scenes/CallingPage.tsx

```diff
@@ -241,10 +256,15 @@ export default function CallingPage({ locale, routePath }: SceneProps) {
     setHasUserDisconnected(true);
   }, [activeConnection]);
 
-  const exitCallingPage = useCallback(() => {
-    disconnectCall();
-    userService?.setShouldSleep(true);
-    contactService?.setActiveContact(null);
+  const handleExitCallButton = useCallback(() => {
+    if (!hasUserDisconnected) {
+      disconnectCall();
+    }
+    if (activeConnection) {
+      openFeedbackDialog();
+    } else {
```

Comment:
Nit: Preferred style for this codebase is to use early return instead of if-else.

Reply:
Oh! Okay :)